Makefile

```diff
@@ -145,30 +145,28 @@ docker-push: docker-build ## Push docker images to docker registry
 
 .PHONY: docker-build-controller
 docker-build-controller: ## Build docker image for NeonVM controller
-	docker build --build-arg VM_RUNNER_IMAGE=$(IMG_RUNNER) --build-arg BUILDTAGS=$(if $(PRESERVE_RUNNER_PODS),nodelete) -t $(IMG_CONTROLLER) -f neonvm/Dockerfile .
+	DOCKER_BUILDKIT=1 docker build --build-arg VM_RUNNER_IMAGE=$(IMG_RUNNER) --build-arg BUILDTAGS=$(if $(PRESERVE_RUNNER_PODS),nodelete) -t $(IMG_CONTROLLER) -f neonvm/Dockerfile .
```

Comment:
Is DOCKER_BUILDKIT=1 really needed? I googled, and it says docker has it by default in the current version (v23.0+)

Reply:
>  Is DOCKER_BUILDKIT=1 really needed? I googled, and it says docker has it by default in the current version (v23.0+)

Debian bookworm comes with an older version:

```
$ docker --version
Docker version 20.10.24+dfsg1, build 297e128
```